larsim/PhotonPropagation/PDFastSimPVS_module.cc

```diff
@@ -3,7 +3,7 @@
 // Plugin Type: producer
 // File:        PDFastSimPVS_module.cc
 // Description:
-// - acts on sim::SimEnergyDeposit from LArG4Main,
+// - acts on sim::SimEnergyDeposit from LArG4Main, 
 // - simulate (fast, photon visibility service) the OpDet response to optical photons
 // Input: 'sim::SimEnergyDeposit'
 // Output: 'sim::OpDetBacktrackerRecord'
```

Comment:
I will rely on others to make provide feedback on the changes here in this file.  my only comment is there seem to be some whitespace-only changes, which make it difficult to review the meaningful changes.  This usually happens when you don't have the latest changes from the `develop` branch.